Roadmap.md

```diff
@@ -33,7 +36,44 @@ Tasks are tracked on a weekly basis through a dedicated spreadsheet.
 For more information see [Confidential Containers V0 Plan](https://docs.google.com/spreadsheets/d/1M_MijAutym4hMg8KtIye1jIDAUMUWsFCri9nq4dqGvA/edit#gid=0&fvid=1397558749).
 
 
-# Mid/long term roadmap 
+# Mid term roadmap
+
+Continue our journey using knowledge and support of SME's in other projects to form stronger 
+opinions on what is needed from components which can be integrated to deliver the confidential 
+containers objectives.
+
+- Harden the code used for the demos, improve CI/CD, release process
+- Establish processes and tools to support planning, prioritisation, and work in progress
+- Simplify the process to get up and running with different underlying Trusted Execution 
+  Environments 
+- Develop a small, simple, secure, lightweight and high performance OCI container image 
+  management library [image-rs](https://github.com/confidential-containers/image-rs) for 
+  confidential containers.
+- Develop small, simple shim firmware ( [td-shim](https://github.com/confidential-containers/td-shim) )
+  in support of trusted execution environment for use with cloud native confidential containers.
+
```

Comment:
I wonder if we should also start building our threat model, and have a clearer documentation for what we're protecting and how we're achieving this.

Reply:
Definately agree, will update